data/channels/test.yml

```diff
@@ -0,0 +1,9 @@
+name: Epicurious
```

Comment:
Not sure if this is an issue, this file is named `test.yml` not something like `epicurious.yml`

Reply:
Woops! that was a test file

Reply:
WHy give it such a cryptic name :D